src/web/src/controllers/Schedule.js

```diff
@@ -245,6 +253,10 @@ class Schedule {
 
       this.selectedSections.push(courseSection);
       console.log(`Added new courseSection ${JSON.stringify(courseSection)}`);
+      if (this.totalCredits()+course.max_credits>21){
```

Comment:
Perhaps we move the number `21` to admin panel. wyt @RichtXO @yangytjason2000?

Reply:
I think we should utilize the type of students in the account when the user is signed in. I guess moving the number to the admin panel would be used for users who aren't logged in. I like that idea.